mlflow/ml-package-versions.yml

```diff
@@ -21,6 +21,10 @@ sklearn:
     run: |
       pytest tests/sklearn/test_sklearn_autolog.py
 
+      # Ensure sklearn autologging works without matplotlib
+      pip uninstall -q -y matplotlib
+      pytest tests/sklearn/test_sklearn_autolog_without_matplotlib.py
```

Comment:
@fa9r Added a test here.

Reply:
It worked: https://github.com/mlflow/mlflow/runs/6689715702?check_suite_focus=true#step:12:253